app/components/view_component/form/rich_text_area_component.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
-module ViewComponent
-  module Form
-    class RichTextAreaComponent < FieldComponent
-      self.tag_klass = ActionView::Helpers::Tags::ActionText
+if defined?(ActionView::Helpers::Tags::ActionText)
```

Comment:
You already depend on the code that would define this:
```ruby
  spec.add_dependency "activesupport", [">= 6.0.0", "< 7.1"]
```

Alternatively ensure that it gets loaded properly, and not rely on whatever app is running to do it for you.  I think that would follow the principle of least surprise.

Rather than doing this, which would change the functionality of the gem unexpectedly, I you could do one of the following:

1. Explain in the readme that people need either
  - `require 'rails/all'`, or
  - `require 'action_text/engine'`
2. Require the discrete file in this library

I am not sure if 2 is feasible.  So perhaps 1 is the best option.

Reply:
I created a new issue for the actual problem:
https://github.com/pantographe/view_component-form/issues/111

Reply:
I've closed #111 because this solution does seem better.  Allow users to decide if they want to actiontext-based features of this gem.  It should be documented that users will have to add the actiontext dependency on their own if they want this to load.  That's not a breaking change as it is *already* the case now (see https://github.com/pantographe/view_component-form/issues/100).